conf/systemd/plackup_ledgersmb.service

```diff
@@ -0,0 +1,33 @@
+# Place this in /etc/systemd/system/plackup_ledgersmb.service; enable & start
+[Unit]
+Description=LedgerSMB Plack
+After=network.target
+
+[Service]
+Restart=always
+User=www-data
+WorkingDirectory=WORKING_DIR
+#
+ExecStart=/usr/bin/plackup                      \
+    --host ubuntu-vm                            \
+    --port 5000                                 \
+    --daemonize                                 \
+    --workers 5                                 \
+    --access-log /var/log/plackup_ledgersmb.log \
+    -I lib                                      \
+    -E development                              \
+    --server HTTP::Server::PSGI                 \
+    -R bin,lib,lib/LedgerSMB/Request            \
+    tools/starman.psgi
+#
+# Select the desired ssrver and adjust the server line in the above configuration
```

Comment:
Spelling errors :-)

Also, can we document that possibly additional packages may be required on a distro before this works? (Is there a way to check for users? Then that'd be a nice addition too.)